File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/fragments/progress/view/ProgressView.java
package ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.progress.view;

import ua.meugen.android.pdfcomposer.model.events.PdfExportProgressEvent;
import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.view.MvpView;


public interface ProgressView extends MvpView {

    void onUpdateProgress(PdfExportProgressEvent event);
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/fragments/progress/view/ProgressView.kt
package ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.progress.view

import ua.meugen.android.pdfcomposer.model.events.PdfExportProgressEvent
import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.view.MvpView


interface ProgressView : MvpView {

    fun onUpdateProgress(event: PdfExportProgressEvent)
}
